### PR TITLE
Documentation overhaul for gh-pages + updates

### PR DESCRIPTION
**Pull request by yamkovoy**
+ Updated documentation accessibility to work better with `gh-pages` and `vitepress` (in comparison to original docs from Confluence).
+ Added the **Notifications** topic.
+ Restructured and updated all of the topics to correlate with the Bokolo PG UI (in comparison to the previous DLak version).
+ Other minor changes to formatting, wording, etc.

Will add some other minor changes tomorrow as a new pull request.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,43 +1,29 @@
...
+## Navigation
 
-As a new bank in the Bokolo network, you must first perform the initial setup. Please follow the instructions in the topics below to successfully set up your bank:
+If you are in need of assistance with any of the UI tabs that are accessible to you as a Bokolo Payment Gateway user of your bank, please select a **Tabs and Screens** topic with the same name from the **Menu**.
+
+The **Menu** is always visible on the left of the page when you are viewing this Help on a PC or another device with a large screen. When viewing from a mobile device, the **Menu** is at the top of the page.
+
+## Initial Setup
+
+If your bank has only joined Bokolo network, you must perform the initial setup.
```
Comment (outoftardis) [inline docs/index.md]:
```suggestion
If your bank has just joined Bokolo network, you must perform the initial setup.
```

File: docs/index.md [hunk 2]
```diff
@@ -1,43 +1,29 @@
...
+
+Please follow the instructions from the **Getting Started** topics below to successfully set up your bank:
```
Comment (outoftardis) [inline docs/index.md:20]:
```suggestion
Please follow the instructions from the **Getting Started** section below to successfully set up your bank:
```

File: docs/tabs-and-screens/notifications.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+To create a new notification, perform the following steps:
+1. On the **Notifications** tab, select **Create new notification** button.\
+   **New notification** window appears.
+2. Depending on who the recipients of your new notification are going to be, do one of the following:
+   - If you intend to send out the notification to all of the Bokolo Mobile users, stay on the **To everyone** filter at the top of the window.
+   - If you intend to send out the notification to specific users, <abbr title="See INFO below">groups of users</abbr> or users of a specific bank, select the **To specific users/groups** filter at the top of the window; then do one of the following:
+     - To send notification to all users of a specific bank, input their **Account ID** into the **Recipient ID(s)** field.
+     - To send notification to a specific user, input either their **Account ID** or any part of their name to look them up and select from the search results.
+
+      > If necessary, perform multiple times for a number of banks and/or users.
```
Comment (outoftardis) [inline docs/tabs-and-screens/notifications.md:36]: why this markup?

File: docs/tabs-and-screens/limits.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+**KYC Type** determines the group of users that the limit is applied to:
+- `FULL_KYC` for verified users;
+- `PARTIAL_KYC` for unverified users.
+
+:::
+
+## Editing an existing limit
+
+::: tip NOTE
```
Comment (outoftardis) [inline docs/tabs-and-screens/limits.md:46]: if it's a note, then use

```suggestion
::: info
```

File: .vitepress/config.mts [hunk 5]
```diff
@@ -30,94 +30,89 @@ export default defineConfig({
         {
           link: "roles",
           text: "Roles and Permissions",
+          items: [],
         },
```
Comment (0x009922) [inline .vitepress/config.mts:34]: Maybe nest this section somewhere else?

Or leave it as:

```suggestion
        {
          text: "Roles and Permissions",
          items: [{ text: 'Overview', link: '/roles' }],
        },
```

Having a sole clickable top-level sidebar item is not consistent with the other sections.

**Review by 0x009922 — APPROVED** [review]

**Pull request merged**